src/components/Speciality/Projects.js

```diff
@@ -4,38 +4,60 @@ import { Row, Col, Grid } from '../grid'
 import { SectionTitle, CardTitle, Subtitle, BodyPrimary } from '../Typography'
 import { Padding } from 'styled-components-spacing'
 import breakpoint from 'styled-components-breakpoint'
+import getColorLuminance from '../../utils/getColorLuminance'
 import { AnimatedLink, CardHeader, PosterImage } from '../Common/animatedLink'
 import LogoGrid from '../Common/LogoGrid'
 
 const Emphasis = styled.em`
   color: ${props => props.theme.colors.secondaryText};
 `
 
-const PosterLinks = ({ project }) => (
-  <AnimatedLink to={`/case-study/${project.slug}`} title={project.title}>
-    <section
-      style={{
-        background: `#${project.posterColor}`
-      }}
-    >
-      <CardHeader>
-        <CardTitle reverse noPadding bigger>
-          {project.title}
-        </CardTitle>
-        <BodyPrimary reverse muted>
-          {project.introSentence.introSentence}
-        </BodyPrimary>
-      </CardHeader>
-      <PosterImage justifyCenter alignCenter color={project.posterColor}>
-        <img
-          alt={project.posterImage.title}
-          src={project.posterImage.file.url}
-          style={{ maxHeight: '100%' }}
-        />
-      </PosterImage>
-    </section>
-  </AnimatedLink>
-)
+const PosterLinks = ({ project }) => {
+  const preferAdditionalPoster =
+    project.slug === 'central-working-changing-how-you-work'
+
+  const poster = {
+    title: preferAdditionalPoster
+      ? project.additionalPoster.title
+      : project.posterImage.title,
+    url: preferAdditionalPoster
+      ? project.additionalPoster.file.url
+      : project.posterImage.file.url,
+    color: project.posterColor
+  }
+
+  const isDarkPosterColor = getColorLuminance(poster.color) < 132.5
```

Comment:
Is there a reason for choosing `132.5` instead of `127.5`?

Reply:
I didn't know 127.5 refers to a specific lightness, 132.4 is Learnerbly luminance. Is 127.5 a special value in terms of luminance?

Reply:
central-working luminance:  `243`
learnerbly luminance: `132.4`

both above `127.5` so I've changed the limit to be 132.5 closer to `learnerbly`.